vite-project-kabinet/src/Content/LoadedList.jsx

```diff
@@ -1,22 +1,78 @@
 import "./paginated.style.css"
 import { CardPreview } from "./CardPreview.jsx"
 import { useState } from "react";
+import { useEffect } from "react";
 
-export const LoadedList = ({items, itemsPerPage}) => {
+export const LoadedList = ({itemsPerPage}) => {
+    const [items, setItems] = useState([])
+    const [totalRecords, setTotalRecords] = useState(0)
     const [visibleCount, setVisibleCount] = useState(itemsPerPage)
 
+    const getPostsWithCategoryName = async (perPage) => {
+
+        const reqPosts = await fetch(`https://casopiskabinet.sk/wp-json/wp/v2/posts?per_page=${perPage}&page=1`)
```

Comment:
Často opakuješ `https://casopiskabinet.sk/wp-json/wp/v2` což může vést k chybám z kopírování (např. zdvojené lomítko na řádku 18 tady v tom souboru) a pracnějším úpravám někdy v budoucnu. Je to zase ideální kandidát na globální constantu.